tests/container_disk_test.go

```diff
@@ -128,6 +130,48 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				}
 			})
 		})
+		It("[Serial]should obey the disk verification limits in the KubeVirt CR", func() {
```

Comment:
I would consider splitting this test into two as you are testing 1. that a VM fails with small limit and 2. that a VM does not fail with our default limit. This will both make the test shorter and clearer and will allow the two to run in parallel.

Reply:
Thanks for the suggestion, I can indeed split the test but these cannot be run in parallel, they'd have to be serial anyway as they modify the KubeVirt CR which would likely impact other tests running in parallel.